frontend/src/components/question-creation/question-elements/TextElement.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+import { Input } from "@chakra-ui/react";
+import QuestionElement from "../QuestionElement";
+
+const TextElement = (): React.ReactElement => {
+  const [text, setText] = useState("");
+
+  return (
+    <QuestionElement>
+      <Input
```

Comment:
right now the text doesn't wrap around to multiline - is there a way we can support that to match figma?
<img width="805" alt="image" src="https://user-images.githubusercontent.com/69697180/220232116-73e43865-c19d-40d5-a95a-f3d432bd323d.png">
<img width="456" alt="image" src="https://user-images.githubusercontent.com/69697180/220232190-3ada0e54-594b-4eac-8e19-df226c25887f.png">